nucleo_wb55rg:samples/bluetooth/peripheral/sample.bluetooth.peripheral fails to build on master

sanitycheck --build-only -p nucleo_wb55rg -T samples/bluetooth/peripheral

produces:

```
[ 46%] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/flash/flash_stm32wbx.c.obj
/mnt/nordic/zp/zephyr/drivers/flash/flash_stm32wbx.c: In function 'flash_stm32_check_status':
/mnt/nordic/zp/zephyr/drivers/flash/flash_stm32wbx.c:178:15: error: 'FLASH_TypeDef' {aka 'struct <anonymous>'} has no member named 'sr'
  178 |  error = (regs->sr & FLASH_FLAG_SR_ERRORS);
      |               ^~
/mnt/nordic/zp/zephyr/drivers/flash/flash_stm32wbx.c:179:16: error: 'FLASH_TypeDef' {aka 'struct <anonymous>'} has no member named 'eccr'
  179 |  error |= (regs->eccr & FLASH_FLAG_ECCC);
      |                ^~
/mnt/nordic/zp/zephyr/drivers/flash/flash_stm32wbx.c:183:7: error: 'FLASH_TypeDef' {aka 'struct <anonymous>'} has no member named 'sr'
  183 |   regs->sr |= FLASH_FLAG_SR_ERRORS;
      |       ^~
make[2]: *** [zephyr/CMakeFiles/zephyr.dir/build.make:674: zephyr/CMakeFiles/zephyr.dir/drivers/flash/flash_stm32wbx.c.obj] Error 1
```

--- a/drivers/flash/flash_stm32wbx.c
+++ b/drivers/flash/flash_stm32wbx.c
@@ -175,12 +175,12 @@ int flash_stm32_check_status(struct device *dev)
 	u32_t error = 0;
 
 	/* Save Flash errors */
-	error = (regs->sr & FLASH_FLAG_SR_ERRORS);
-	error |= (regs->eccr & FLASH_FLAG_ECCC);
+	error = (regs->SR & FLASH_FLAG_SR_ERRORS);
+	error |= (regs->ECCR & FLASH_FLAG_ECCC);
 
 	/* Clear systematic Option and Enginneering bits validity error */
 	if (error & FLASH_FLAG_OPTVERR) {
-		regs->sr |= FLASH_FLAG_SR_ERRORS;
+		regs->SR |= FLASH_FLAG_SR_ERRORS;
 		return 0;
 	}
 

